css/css-fonts/variations/font-shorthand.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Testing font shorthand for new values introduced in CSS Fonts level 4</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <div id="shorthand-test">Shorthand test</div>
+
+    <script>
+        testFontShorthand = [
+            { value: "calc(24px) Arial",                                     isValid:true,   message: "Font weight specified as number" },

```

Comment:
Comment is incorrect. This should be something like "font-weight is not part of shorthand"